project/app/api/visualization.py

```diff
@@ -0,0 +1,44 @@
+import logging
+
+from fastapi import APIRouter
+
+from app.utils.security.header_checking import AuthRouteHandler
+from app.api.models import HistogramRequest, LineGraphRequest
+from app.utils.visualizations import histogram, line_graph
+
+# global variables and services
+router = APIRouter(route_class=AuthRouteHandler)
+log = logging.getLogger(__name__)
+
+
+@router.route("/viz/linegraph")
+def return_line_graph(data: LineGraphRequest):
+    """Endpoint produces a line graph of a student's squad scores over time
```

Comment:
Can you add a quick clarifier to both visualization functions that this will only be the student's Squad Scores from "accepted" or "moderator-approved" stories?